pcsx2/ImGui/FullscreenUI.cpp

```diff
@@ -22,6 +22,13 @@
 #include "USB/USB.h"
 #include "VMManager.h"
 #include "ps2/BiosTools.h"
+#include "DEV9/AdapterUtils.h"
```

Comment:
Doesn't look like this include is necessary?

Reply:
yup nope it's not I mistakenly added it (it's gone now as of latest push)